app/forms/flood_risk_engine/steps/base_postcode_form.rb

```diff
@@ -0,0 +1,20 @@
+module FloodRiskEngine
+  module Steps
+    class BasePostcodeForm < BaseRedirectableForm
+
+      property :post_code, virtual: true
```

Comment:
its virtual and purposelessly named as to be different to address. this form is not backed by a model